docs/reference/installation.rst

```diff
@@ -7,7 +7,8 @@ Installation
 Prerequisites
 -------------
 
-PHP 7 and Symfony 2.8 or 3 are needed to make this bundle work; there are also some Sonata dependencies that need to be installed and configured beforehand:
+PHP 7 and Symfony 2.8 or 3 are needed to make this bundle work, there are
```

Comment:
7.1. Symfony 2.8, >=3.2 or 4 right?

Reply:
3.2 support is ending next week, should I mark it as 3.3 instead?